fission/package.json

```diff
@@ -41,7 +41,7 @@
     "react-colorful": "^5.6.1",
     "react-dom": "^18.3.1",
     "react-icons": "^4.12.0",
-    "three": "^0.177.0",
+    "three": "^0.178.0",
```

Comment:
@Dhruv-0-Arora the bun.lock file was actually out of date, the version (since #1164) has been `0.177.0`, I checked the 177 -> 178 changelog and there's nothing relevant for us in it. This PR is fixing issues caused by the security-motivated version bump that already happened